ClientManagementService/ClientManagementService.Infrastructure/Persistence/PetRepository.cs

```diff
@@ -0,0 +1,104 @@
+﻿using ClientManagementService.Infrastructure.Persistence.Entities;
+using ClientManagementService.Infrastructure.Persistence.Filters.Pet;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ClientManagementService.Infrastructure.Persistence
+{
+    public interface IPetRepository
+    {
+        Task AddPet(Pet newPet);
+        Task DeletePetById(long petId);
+        Task<List<Pet>> GetAllPets();

```

Comment:
If we do this, we need to do paging like we do users.

If we have 100,000 pets in our db, that will be a large payload we return using HTTP. And it will take a long time.

Reply:
this has not been addressed